jetty-http/src/main/java/org/eclipse/jetty/http/HttpStatus.java

```diff
@@ -133,7 +133,9 @@
         NOT_MODIFIED(NOT_MODIFIED_304, "Not Modified"),
         USE_PROXY(USE_PROXY_305, "Use Proxy"),
         TEMPORARY_REDIRECT(TEMPORARY_REDIRECT_307, "Temporary Redirect"),
+        // Keeping the typo for backward compatibility for a while
```

Comment:
Good change.   Shows that this PR wasn't 100% automated.

Reply:
Yes, I've visually reviewed all the detections and applied only necessary ones. 😄